expand-contract-upgrade/validate.sh

```diff
@@ -133,6 +133,9 @@ validate_nodes() {
 }
 
 validate_elasticsearch() {
+  # give time for master/client/data nodes
+  sleep 60
```

Comment:
we need to pole instead of sleeping.  We need to get @homernajafi PR in.  It fixed a lot of this.